.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
dist should included in the  .gitignore , that folder is the build output kinda like the target directory in java

Reply:
Made requested change.